site/quickstart.md

```diff
@@ -246,8 +252,10 @@ $ curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/
  < Vary: Accept-Encoding
 < * Connection #0 to host your.server.com left intact
 ```
-**Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or
-update security lists to allow ingress to this port.
+
+**Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or update security lists to allow ingress to this port.
+
+f.	To access the WLS Administration Console, edit the `my-inputs.yaml` file (assuming that you named your copy `my-inputs.yaml`) to set `exposedAdminNodePort: true`. Open a browser to `http://your.server.com:30305`. As in the previous step, substitute the Node IP address of the worker node for `your.server.com`.
```

Comment:
The port is 30701 not 30305

Reply:
Changed all references from 30305 to 30701.